example/ios/Podfile

```diff
@@ -2,7 +2,7 @@ source 'https://cdn.cocoapods.org/'
 source 'https://github.com/m0nac0/flutter-maplibre-podspecs.git'
 
 # Uncomment this line to define a global platform for your project
-platform :ios, '9.0'
+platform :ios, '11.0'
```

Comment:
Are there any specific reasons for the changes in the example/ios folder?

Reply:
Or were these changes generated automatically?

Reply:
Yes, I executed the example app with flutter 3.10.1 and it generates those and a few other changes.